popSimManu.tex

```diff
@@ -519,68 +519,73 @@ \subsection*{Use case: comparing methods of demographic inference}
 \section*{Discussion}
 
 Here we have described what we view as the first product from the PopSim Consortium:
-the \stdpopsim library. We have founded the Consortium with a number of specific goals
-in mind: standardization of simulation within the computational genetics community,
-increasing reproducibility of simulation based results, community-based development and
-decision making in guiding best practices in population genetics, and eventual benchmarking
-of inferential methods.
+the \stdpopsim library. We have founded the Consortium with a number of specific goals in mind:
+standardization of simulation within the computational genetics community,
+increasing reproducibility and ease of use of complex simulation based results,
+community-based development and decision making guiding best practices in population genetics,
+and eventual benchmarking of inferential methods.
 
 The \stdpopsim library is the first piece of this effort, in that it allows for rigorous
 standardization of complex population genetic simulations. Population genetics, as a field,
-% RNG: We should have more coalescence puns in the paper! :-P
-has yet to coalesce around a set of gold-standards for the crucial task of method
+has yet to coalesce around a set of standards for the crucial task of method
 evaluation, which in our field hinges on simulation. In contrast, other fields such as
 structural biology \citep{moult1995large} and machine learning \citep{russakovsky2015imagenet} have a long track record
 of standardized method testing. We hope that our efforts represent the beginning of what
 will prove to be an equally longstanding and invaluable tradition in population genetics.
 
-Moreover individual research groups are often in the position of having to independently
-reimplement complex previously published models. \stdpopsim offers solutions to both of
-these issues by creating a quality controlled, easy-to-use, gateway to models and organisms
+Individual research groups are often in the position of having to independently
+reimplement complex previously published demographic models.
+Additional layers of realism (e.g., recombination maps) makes this especially daunting.
+\stdpopsim aims to lower this barrier
+by creating a quality controlled, easy-to-use, gateway to models and organisms
 that are central to modern research in genetics.
```

Comment:
trying to use the phrase 'lower the barrier'

Reply:
we are missing a comma between "complex" and "previously published"

Reply:
not sure I agree but either way